src/components/BeverageDetailsLoader/LoadedBeverageDetails.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import type { Beverage } from 'brewskey.js-api';
+
+import * as React from 'react';
+import { Dimensions, Text, StyleSheet, View } from 'react-native';
+import SectionHeader from '../../common/SectionHeader';
+import CacheImage from '../../common/CachedImage';
+import CONFIG from '../../config';
+import { TYPOGRAPHY } from '../../theme';
+import OverviewItem from '../../common/OverviewItem';
+
+const { width: screenWidth } = Dimensions.get('window');
+const BEVERAGE_IMAGE_HEIGHT_COEFFICIENT = 0.75;
+
+const styles = StyleSheet.create({
+  descriptionText: {
+    ...TYPOGRAPHY.paragraph,
+    paddingHorizontal: 15,
+  },
+});
+
+type Props = {|
+  value: Beverage,
```

Comment:
Since this isn't a generic component why not just call this `beverage`?

Reply:
because I pass it as `<LoaderComponent loadedComponent={LoadedBeverageDetails} />` - to rename it I need to use some hoc like `renameProps()` which I don't have atm